src/routes/Project/actions/TableActionHandlers.js

```diff
@@ -0,0 +1,13 @@
+import ProjectConstants from '../constants';
+
+const ACTION_HANDLERS = {
+  [ProjectConstants.ADD_CLIP_TO_TRACK]: (state, action) => {
+    const id = action.payload.id;
+    const track = action.payload.start.y;
+    const startTime = action.payload.start.x;
+    const endTime = action.payload.end.x;
+    return { ...state,  [track]: { ...state[track], [startTime]: id, [endTime]: id } };
```

Comment:
this is kind of an odd way of organizing this, but it kind of makes sense. any particular reason you decided to do it this way?